### ISSUE TITLE
Invalid Input Provided [Integration email cannot be empty.]}

### ISSUE THREAD

**Issue by nserebry** [terraform-providers/terraform-provider-pagerduty#9]
Hi ,
Maybe an easy question, but I'm relatively new to terraform. 
Please help to resolve below error with Zendesk resource pagerduty_service_integration

### Terraform Version
Terraform v0.9.8

### Affected Resource(s)

Generated after terraform apply command 
```
data "pagerduty_vendor" "zendesk" {
  name = "zendesk"
}

resource "pagerduty_service_integration" "PartnerAssistance-zendesk" {
  name    = "${data.pagerduty_vendor.zendesk.name}"
  service = "${pagerduty_service.PartnerAssistance.id}"
  vendor  = "${data.pagerduty_vendor.zendesk.id}"
}
```

### Debug Output

```
Error applying plan:

1 error(s) occurred:

* pagerduty_service_integration.PartnerAssistance-zendesk: 1 error(s) occurred:

* pagerduty_service_integration.PartnerAssistance-zendesk: Failed call API endpoint. HTTP response code: 400. Error: &{2001 Invalid Input Provided [Integration email cannot be empty.]}

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

### Expected Behavior
 I'm not sure where do I add the email in my JInja2  template to get rid of this message ` Integration email cannot be empty` ?

### Steps to Reproduce
1. Creating PD integration with Jinja2. Everything works fine, except zendesk integration. 
```
resource "pagerduty_service_integration" "{{ team }}-{{ vendor }}" {
  name    = "${data.pagerduty_vendor.{{ vendor }}.name}"
  service = "${pagerduty_service.{{ team }}.id}"
  vendor  = "${data.pagerduty_vendor.{{ vendor }}.id}"
}
```
2 . After I ran `terraform apply`  everything ran just fine. But seems like zendesk integration is failed.  But I can't find how to add email to the template.  No example here as well https://www.terraform.io/docs/providers/pagerduty/r/service_integration.html

Thank you, 
N

**Comment by heimweh** [comment]
Hi @nserebry,

thanks for submitting this issue and sorry for the issue you're experiencing.

I just replicated this and the error message is far from ideal.
It seems like this particular integration requires the field `integration_email` to be set. 
The integration email must also end with the account domain (e.g `@acme.pagerduty.com`).

Could you try setting this field and see if that solves the issue you're experiencing?

Best,

Alexander

**Comment by nserebry** [comment]
@heimweh thank you, that helped.

**Comment by heimweh** [comment]
@nserebry glad that it works!
 I'm gonna go ahead and close this one for now, but feel free to re-open this at any time!

**Comment by 2rs2ts** [comment]
Please update the docs to specify that `integration_email` is an optional input field not just an output attribute. This bit us today as well with the generic email integration.

**Comment by heimweh** [comment]
Hi @2rs2ts, thanks for reporting this.

I've published a fix here: #36. 

Many thanks,

Alexander
